### DISCUSSION TITLE
On the Google Collab Notebook

### DISCUSSION THREAD

**Discussion by SDPDVADER** [TheLastBen/fast-stable-diffusion#1424] [General]
suddenly the started to happen
![image](https://user-images.githubusercontent.com/123031699/213311955-4fae7d28-29f1-422a-b38f-ca2d70c4cca5.png)

There is no longer a view option when running prompts but they do save in the drive.  Please advise.  

After it runs, then also the Generate button is frozen until you refresh. so that is going on.

**Comment by TheLastBen** [comment]
this doesn't look like colab, what platform are you using

**Comment by SDPDVADER** [comment]
No sir it is your google collab 

Warning: unknown mime-type for "outputs/img2img-images" -- using "application/octet-stream" Error: no "view" mailcap rules found for type "application/octet-stream" /usr/bin/xdg-open: 869: www-browser: not found /usr/bin/xdg-open: 869: links2: not found /usr/bin/xdg-open: 869: elinks: not found /usr/bin/xdg-open: 869: links: not found /usr/bin/xdg-open: 869: lynx: not found /usr/bin/xdg-open: 869: w3m: not found xdg-open: no method available for opening 'outputs/img2img-images' Warning: unknown mime-type for "outputs/img2img-images" -- using "application/octet-stream" Error: no "view" mailcap rules found for type "application/octet-stream" /usr/bin/xdg-open: 869: www-browser: not found /usr/bin/xdg-open: 869: links2: not found /usr/bin/xdg-open: 869: elinks: not found /usr/bin/xdg-open: 869: links: not found /usr/bin/xdg-open: 869: lynx: not found /usr/bin/xdg-open: 869: w3m: not found xdg-open: no method available for opening 'outputs/img2img-images' Warning: unknown mime-type for "outputs/img2img-images" -- using "application/octet-stream" Error: no "view" mailcap rules found for type "application/octet-stream" /usr/bin/xdg-open: 869: www-browser: not found /usr/bin/xdg-open: 869: links2: not found /usr/bin/xdg-open: 869: elinks: not found /usr/bin/xdg-open: 869: links: not found /usr/bin/xdg-open: 869: lynx: not found /usr/bin/xdg-open: 869: w3m: not found xdg-open: no method available for opening 'outputs/img2img-images' Warning: unknown mime-type for "outputs/img2img-images" -- using "application/octet-stream" Error: no "view" mailcap rules found for type "application/octet-stream" /usr/bin/xdg-open: 869: www-browser: not found /usr/bin/xdg-open: 869: links2: not found /usr/bin/xdg-open: 869: elinks: not found /usr/bin/xdg-open: 869: links: not found /usr/bin/xdg-open: 869: lynx: not found /usr/bin/xdg-open: 869: w3m: not found xdg-open: no method available for opening 'outputs/img2img-images' Warning: unknown mime-type for "outputs/img2img-images" -- using "application/octet-stream" Error: no "view" mailcap rules found for type "application/octet-stream" /usr/bin/xdg-open: 869: www-browser: not found /usr/bin/xdg-open: 869: links2: not found /usr/bin/xdg-open: 869: elinks: not found /usr/bin/xdg-open: 869: links: not found /usr/bin/xdg-open: 869: lynx: not found /usr/bin/xdg-open: 869: w3m: not found xdg-open: no method available for opening 'outputs/img2img-images' Warning: unknown mime-type for "outputs/img2img-images" -- using "application/octet-stream" Error: no "view" mailcap rules found for type "application/octet-stream" /usr/bin/xdg-open: 869: www-browser: not found /usr/bin/xdg-open: 869: links2: not found /usr/bin/xdg-open: 869: elinks: not found /usr/bin/xdg-open: 869: links: not found /usr/bin/xdg-open: 869: lynx: not found /usr/bin/xdg-open: 869: w3m: not found xdg-open: no method available for opening 'outputs/img2img-images' Error completing request Arguments: ('', [], False, -1) {} Traceback (most recent call last): File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/modules/call_queue.py", line 56, in f res = list(func(*args, **kwargs)) File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/modules/ui.py", line 116, in save_files data = json.loads(js_data) File "/usr/lib/python3.8/json/__init__.py", line 357, in loads return _default_decoder.decode(s) File "/usr/lib/python3.8/json/decoder.py", line 337, in decode obj, end = self.raw_decode(s, idx=_w(s, 0).end()) File "/usr/lib/python3.8/json/decoder.py", line 355, in raw_decode raise JSONDecodeError("Expecting value", s, err.value) from None json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)

**Comment by SDPDVADER** [comment]
From <https://colab.research.google.com/github/TheLastBen/fast-stable-diffusion/blob/main/fast_stable_diffusion_AUTOMATIC1111.ipynb?authuser=3#scrollTo=PjzwxTkPSPHf>

**Comment by TheLastBen** [comment]
how to reproduce this error ?

**Comment by SDPDVADER** [comment]
The first prompts works fine and the image viewer shows them, but when you try to do it again it stays on the old one and then the Generate button becomes frozen. it is almost like it just staying on the first images only and not producing on the screen but the images are created and saved on the drive.

**Comment by SDPDVADER** [comment]
if there was an update then something caused this to happen because it was working fine yesterday.

**Comment by SDPDVADER** [comment]
could DDIM be the culprit?

**Comment by TheLastBen** [comment]
are you using gradio server or localtunnel ?

**Comment by SDPDVADER** [comment]
I was using both.  Either the image memory is too much or  something with the browser.  It seems like it gets in cycle and doesn't allow the buttons to be pressed.

**Comment by SDPDVADER** [comment]
For some reason Automatic1111's not even giving me the output image to my UI window over half the time suddenly, and I haven't updated it, nor has Windows updated, or indeed have I updated ANYTHING related to it like firefox or whatever, so I'm a little baffled at this new problem popping up since I can't think of what I could possibly have changed to cause this issue to pop up. It finishes the render fine, I can view it in my folder, but it's not putting it in the UI and just sticking on the render before. It's not a consistent issue, that doesn't seem to be a set pattern to when it'll actually update the UI when finishing render vs just sending it to my outputs folder.

**Reply by TheLastBen** [reply]
Use the gradio server and see if the issue persists

**Comment by bjornlarssen** [comment]
I have the same problem. Enabled warnings in settings. This is what happens:

Warning: unknown mime-type for "outputs/txt2img-images" -- using "application/octet-stream"
Error: no "view" mailcap rules found for type "application/octet-stream"
/usr/bin/xdg-open: 869: www-browser: not found
/usr/bin/xdg-open: 869: links2: not found
/usr/bin/xdg-open: 869: elinks: not found
/usr/bin/xdg-open: 869: links: not found
/usr/bin/xdg-open: 869: lynx: not found
/usr/bin/xdg-open: 869: w3m: not found
xdg-open: no method available for opening 'outputs/txt2img-images'

And then the Generate button does nothing until I reload UI.

I am running a Colab with the latest version. Actually deleted and reinstalled everything last night. The first batch of images is being created (I see the previews), then get the blank image icon, the above warnings, and nothing shows up. The generated images are saved in txt2img-images, I can look at them on the Google Drive. I'm using the Gradio link.

Edit: just deleted everything and reinstalled again, it worked twice instead of once, but then showed me the first generated grid. Now the Generate button does nothing again. There is no error, the images got generated, they are in the outputs folder.

**Reply by TheLastBen** [reply]
A1111 colab or Dreambooth colab ?

**Reply by bjornlarssen** [reply]
A1111.

**Reply by TheLastBen** [reply]
fixed: https://github.com/TheLastBen/fast-stable-diffusion/commit/c3f2a0e4fbdebefdba70bcde98004b6564f3d27b use the latest colab

**Reply by bjornlarssen** [reply]
Thanks so much, it works now!

I'd like to stop WebUI from updating itself – keep the version that works. This is what I found on GitHub:

Remove git pull from webui-user.bat in case it's there.
use --skip-install in your command line arguments.

Should I do the same in the notebook? Is it enough to remove "!git pull" from the Install/Update section and add the argument?

**Reply by TheLastBen** [reply]
you can now remove git pull without facing issues, also the dreambooth testing cell is a frozen repo, it's best to keep git pull there
